lua/rustaceanvim/executors/background.lua

```diff
@@ -38,7 +37,7 @@ M.execute_command = function(command, args, cwd, opts)
       return
     end
     local output = (sc.stderr or '') .. '\n' .. (sc.stdout or '')
-    local diagnostics = require('rustaceanvim.test').parse_diagnostics(fname, output, opts.bufnr)
+    local diagnostics = require('rustaceanvim.test').parse_cargo_test_diagnostics(output, opts.bufnr)
```

Comment:
todo: We should probably check if `args[1]` is `test` or `nextest` and use the appropriate parser here.

Reply:
`try_nextest_transform` is never called in this function, so I figured it would always be `cargo test` (?)

unsure how this function gets called to be honest lol 🙈